### PR TITLE
Bulk withdrawal refactor

### PR DESCRIPTION
**Pull request by constanzauanini**
This is just a refactor of what we had. Created new components to allow re-using them for single and bulk withdrawals.
Bulk withdrawal is not working yet. Only first step of single withdrawal is working.

### CODE REVIEW

**Review by karthikbtf — COMMENTED** [review]
This is great!!! A few comments.

File: src/components/Inventory/withdraw/BulkWithdraw.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from 'react';
+import { ServerOrganization } from 'src/types/Organization';
+import useQuery from 'src/utils/useQuery';
+import SpeciesBulkComponent from './SpeciesBulkComponent';
+import BatchesBulkComponent from './BatchesBulkComponent';
+
+type BulkWithdrawProps = {
+  organization: ServerOrganization;
+};
+export default function BulkWithdraw(props: BulkWithdrawProps): JSX.Element | null {
```
[the remark this answers is not in the document]
Comment (constanzauanini) [reply src/components/Inventory/withdraw/BulkWithdraw.tsx:10]: Oh, if we have two urls, now I can do this suggestion. With same url, I had to point to some component first, that was why I added this one.

File: src/components/Inventory/view/InventorySeedlingsTable.tsx [hunk 2]
```diff
@@ -179,7 +179,8 @@ export default function InventorySeedslingsTable(props: InventorySeedslingsTable
     if (fromColumn === 'withdraw') {
       if (trackingEnabled) {
         history.push({
-          pathname: APP_PATHS.BATCH_WITHDRAW.replace(':batchId', batch.id.toString()),
+          pathname: APP_PATHS.BATCH_WITHDRAW,
+          search: `?batchId=${batch.id.toString()}`,
```
[the remark this answers is not in the document]
Comment (constanzauanini) [reply src/components/Inventory/view/InventorySeedlingsTable.tsx:183]: The search parameter?

[the remark this answers is not in the document]
Comment (karthikbtf) [reply src/components/Inventory/view/InventorySeedlingsTable.tsx:183]: yes, just curious. this is fine as-is.

[the remark this answers is not in the document]
Comment (constanzauanini) [reply src/components/Inventory/view/InventorySeedlingsTable.tsx:183]: No, it can't, it's a string

**Review by karthikbtf — APPROVED** [review]
looks good! thanks so much!
one comment on exporting from an index.ts

**Pull request merged**